API/UploadSlideBacktranslation.php

```diff
@@ -72,16 +71,24 @@ function InitializeNewStory($conn, $androidId, $templateTitle) {
     $slideIndex = 0;
 
     error_log("Copying template '$templateTitle' folder to new project directory and creating slides");
-    foreach ($slideEntries as $slideEntry) {
 
-        if ($slideEntry['slideType'] !== 'COPYRIGHT') {
-        error_log("Got slide number $slideIndex");
-        PrepareAndExecute($conn,
-            'INSERT IGNORE INTO Slide (storyId, note, slideNumber, isApproved) VALUES (?,"",?,0)', 
-            array($storyId, $slideIndex));
+    // Check if storyId already exists in Slide
```

Comment:
How is it possible that we have existing slides for the story? `if (($row = $checkExistingStoryStmt->fetch(PDO::FETCH_ASSOC))) {` checks to make sure that no story with the same phone and title exists. 

`PrepareAndExecute($conn, 'INSERT INTO Stories (title, language, projectId, note) SELECT ?,?,?,""', array($templateTitle, $language, $projectId));` then creates a new story id. By the time we get to this area, the story id should be a brand new id that has never been used. What's the sequence of inputs such that we have some slides for this story id? 

Seems like it's much more likely that slides were from a previous test run and weren't cleaned up.